src/smc-webapp/r_misc/loading.tsx

```diff
@@ -2,13 +2,15 @@ import * as React from "react";
 import { Icon } from "./icon";
 import { TypedMap } from "../app-framework/TypedMap";
 
+export type Estimate = TypedMap<{
+  time: number; // Time in seconds
+  type: "new" | "ready" | "archived";
+}>;
+
```

Comment:
this is the `LoadingEstimate`

Reply:
Wow. Drag to highlight comment region. I didn't know this was possible on GitHub.